examples/sensors/main.c

```diff
@@ -37,31 +41,52 @@ static void timer_fired(__attribute__ ((unused)) int arg0,
   int ninedof_accel_x = 0, ninedof_accel_y = 0, ninedof_accel_z = 0;
   int ninedof_magneto_x = 0, ninedof_magneto_y = 0, ninedof_magneto_z = 0;
   int ninedof_gyro_x = 0, ninedof_gyro_y = 0, ninedof_gyro_z = 0;
+  unsigned proxReading = 0;
 
   /* *INDENT-OFF* */
-  if (isl29035)     ambient_light_read_intensity_sync(&light);
-  if (tmp006)       tmp006_read_sync(&tmp006_temp);
-  if (tsl2561)      tsl2561_lux = tsl2561_get_lux_sync();
-  if (lps25hb)      lps25hb_pressure = lps25hb_get_pressure_sync();
-  if (temperature)  temperature_read_sync(&temp);
-  if (humidity)     humidity_read_sync(&humi);
+  if (isl29035)
+    ambient_light_read_intensity_sync(&light);
+  if (tmp006)
+    tmp006_read_sync(&tmp006_temp);
+  if (tsl2561)
+    tsl2561_lux = tsl2561_get_lux_sync();
+  if (lps25hb)
+    lps25hb_pressure = lps25hb_get_pressure_sync();
+  if (temperature)
+    temperature_read_sync(&temp);
+  if (humidity)
+    humidity_read_sync(&humi);
+  if (proximity)
+    proximity_read_sync(&proxReading);
```

Comment:
Did the tooling make this formatting change automatically? I don't think it should have?

Reply:
Sorry about that.  I think the VSCode C/C++ extension might have cause that but I have reverted this change.